### PR TITLE
INI Changes

### PR DESCRIPTION
**Pull request by nf6429**
This PR deals with more Texture Cache Accuracy cases, and some Dualcore ones too. Might add a commit with bounding box fixes for demo discs, but I've only been able to test US versions, not PAL versions, so I can only safely do discs 20-22 and 35, not 17-19. For the PAL version I do have, the games don't match, so probably best to leave them alone for another PR.

### CODE REVIEW

File: Data/Sys/GameSettings/SAQ.ini [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# SAQE5G - Harley Pasternak's Hollywood Workout
+
+[Video_Settings]
+# Fixes game text.
+SafeTextureCacheColorSamples - 0
```
Comment (Rumi-Larry) [inline Data/Sys/GameSettings/SAQ.ini:5]: This should be a "=" not a "-"

Comment (nf6429) [reply Data/Sys/GameSettings/SAQ.ini:5]: Well spotted.

**Comment by JMC47** [comment]
Can you try using texture cache value 2048 instead of 0 in some of the games?  It's less performance intensive than 0 but more accurate than 512, so it may result in a net gain in performance while also being accurate enough to produce correct emulation.

The Dualcore changes are in titles that make sense to me.

**Comment by nf6429** [comment]
> Can you try using texture cache value 2048 instead of 0 in some of the games? It's less performance intensive than 0 but more accurate than 512, so it may result in a net gain in performance while also being accurate enough to produce correct emulation.
> 
> The Dualcore changes are in titles that make sense to me.

I could, but main concern is how do I test which titles work with 2048, since 0= safe, 512 = middle, 128 = fast, how do i test it in dolphin?

**Comment by JosJuice** [comment]
You have to type 2048 into the INI file and then test.

**Comment by nf6429** [comment]
Had some issues with previous computer I was doing this on, and I'm not particularly good enough at git bash to do this PR in one commit all squashed (normally use git desktop), so I'll likely close this PR and create a new one when the changes are ready on another computer. Note that I tested the 2048 changes, 11 titles work with it.

File: Data/Sys/GameSettings/SW6.ini [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# SW6P78, SW6E78 - WWE'12
+
+[Video_Settings]
+# Fixes Create a Superstar invisible items.
```
Comment (Rumi-Larry) [inline Data/Sys/GameSettings/SW6.ini:4]: I would place the "Create a Superstar" phrase between quotes so it's more readable. Ditto on the othher WWE games.

Comment (nf6429) [reply Data/Sys/GameSettings/SW6.ini:4]: That makes sense, thanks.

**Comment by Avasam** [comment]
@nf6429 
> [...] I'm not particularly good enough at git bash to do this PR in one commit all squashed (normally use git desktop),[...]

`git rebase -i HEAD^X` where X is the number of commits you want to rebase. The `-i` flag is for an interactive rebase. It will use your prefered editor configured in git (defaults to Vim iirc). You can configure it to use whatever tool you prefer (like Notepad++, or VSCode).

Speaking of VSCode, since you seem to prefer using Git with a GUI, it has a nice looking rebase window. Here's how to configure git to use VSCode: https://code.visualstudio.com/docs/editor/versioncontrol#_vs-code-as-git-editor . VSCode + GitLens extension has completely replace any other git tools for me.
And here's for Notepad++: `git config --global core.editor "'C:/Program Files/Notepad++/notepad++.exe' -multiInst -notabbar -nosession -noPlugin"`

**Comment by JosJuice** [comment]
HEAD means the commit you are currently at, and HEAD^X means the commit that's X commits before the current commit. The X is just a placeholder, you have to replace it with a number.

**Comment by nf6429** [comment]
@JosJuice Don't want to bother you, but this is my first time rebasing in git, and as you can see commit wise, its very weird. All I did in the interactive rebase was squash fix spelling error into INI changes and its made 4 commits. I have no idea what I have done

**Comment by nf6429** [comment]
I've got the branch ready for a new PR, I'll close this one since the history is messed up from me figuring out git.

**Pull request closed without merging**